### DISCUSSION TITLE
Problema esposizione Script su google Home

### DISCUSSION THREAD

**Discussion by provaprova1** [indomus/forum#77] [HUB: Home Assistant]
Ciao
ho eseguito la procedura per linkare Home Assistant a Google home
Ora sto provando a usare gli script creati (e che vedo disponibili su google home) tramite routine ma ricevo sempre un messaggio di errore "Impossibile collegarsi ai servizi"
In realtà per qualche ora ha funzionato poi, apparentemente senza motico (non ho fatto alcuna modifica) ha smesso

Questa la mia configurazione

google_assistant:
    project_id: myproject
    service_account: !include SERVICE_ACCOUNT.json
    report_state: true
    exposed_domains:
    - script
    - scene
    - switch

![immagine](https://user-images.githubusercontent.com/31503604/113286034-f0366a80-92eb-11eb-863c-646dd364bb93.png)

Riuscite ad aiutarmi?

**Comment by snakuzzo** [comment]
Se ha funzionato e all'improvviso non va più non è un problema di configurazione (**che comunque ti chiederei di postare con il markdown...non so più come ripeterlo**).

Più che altro dovresti provare a vedere se viene loggato qualocsa su HA quando provi a fare qualcosa su Google Assistant.. In base a quello che hai postato è impossibile dire da cosa possa dipendere

**Reply by provaprova1** [reply]
Ciao, chiedo venia per la formattazione non adeguata
Dove posso trovare  i log di cui parli? Nel file home-assistant.log non c'è nulla relativo a google assistant

**Reply by snakuzzo** [reply]
Sì intendevo quel log. Se ti dà quell'errore e non logga niente, vuol dire che alla tua istanza di Home Assistant nemmeno ci arriva.
Ti suggerirei di ricontrollare tutto il giro (integrazione GCP inclusa)
